pallets/gear-builtin/src/bls12_381.rs

```diff
@@ -57,10 +57,15 @@ impl<T: Config> BuiltinActor for Actor<T> {
 
         (
             result.map(|response| {
-                response
-                    .encode()
-                    .try_into()
-                    .unwrap_or_else(|_| unreachable!("Response message is too large"))
+                response.encode().try_into().unwrap_or_else(|err| {
+                    let err_msg = format!(
+                        "Actor::handle: Response message is too large. \
+                            Got error - {err:?}"
+                    );
```

Comment:
an example of required data for the log is given here
https://github.com/gear-tech/gear/blob/e917669ef0079f8e54fa143f446275a56eb96e61/pallets/gear/src/manager/task.rs#L160

Reply:
FIxed